### PR TITLE
Fix oversights regarding namespace and ports in templates

### PR DESCRIPTION
**Pull request by clamydo**
I've fixes some minor oversights when setting the namespace and the port in the chart templates. Parameterised them - I hope I've done it correctly.

### CODE REVIEW

File: charts/prefect-orion/templates/api/deployment.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
           imagePullPolicy: {{ .Values.api.image.pullPolicy }}
           command: ["prefect", "orion", "start", "--host", "0.0.0.0", "--log-level", "WARNING"]
           ports:
-          - containerPort: 4200
+          - containerPort: {{ .Values.api.service.port }}
```
Comment (jamiedick) [inline charts/prefect-orion/templates/api/deployment.yaml:39]: I dont see the api service port in the values file -- can you add in and perhaps set the default to 4200?

**Review by jamiedick — CHANGES_REQUESTED** [review]
minor change requested - otherwise awesome work!

File: charts/prefect-orion/templates/api/deployment.yaml (hunk 1 above)
Comment (clamydo) [reply charts/prefect-orion/templates/api/deployment.yaml:39]: Hey, glad to help. 

Isn't it? https://github.com/PrefectHQ/prefect-helm/blob/26dca12e31fae629bb3d4ffc0af1bbdc0cffbb77/charts/prefect-orion/values.yaml#L7

**Review by jamiedick — APPROVED** [review]
🚀

**Pull request merged**